task_5/Toma.Lobanova/src/main/java/ua/od/hillel/SequenceTask2/Sequence.java

```diff
@@ -0,0 +1,10 @@
+package ua.od.hillel.SequenceTask2;
+
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+
+public class Sequence {
+    public static void main(String... args){
+        Stream.iterate(1, item -> item + 2).limit(55).peek(item -> System.out.println(item)).collect(Collectors.toList());
```

Comment:
Yes, thank you, i will read more about it. I just took the article and there were many examples with collect. This new subject is very difficult but very interesting as well. I really want to learn to use it and that's why i use it everywhere now)))